### ISSUE TITLE
Wallet should say what currency symbol is "$"

### ISSUE THREAD

**Issue by Transisto** [BlueWallet/BlueWallet#4259]
User can mistaken a $ CAD for $ USD.  Only using $ symbol is not descriptive enough.

Suggestion, Use  MXN, USD or CAD instead of only $

A further feature improvement would be to do like Mycelium, letting user select as many fiat currency to circle through, Also support precious metals and commodities as units. 

To stay compatible we could enable both SAT and BTC by default, that would allow users tick one off if they prefer to only see it in SAT

**Comment by marcosrdz** [comment]
The app supports 1 Fiat unit at a time. The user shouldn’t face any confusion with this as the app will default to the user’s device settings unless manually modified to use a different fiat. @ncoelho what do u think?

**Comment by Transisto** [comment]
Canadian constantly have to transact in USD or CAD and it's almost impossible to remember what currency the wallet is in when using it about once a week.

**Comment by Overtorment** [comment]
perhaps, CAD should be a special case and handled differently?

**Comment by Transisto** [comment]
> perhaps, CAD should be a special case and handled differently?

Huh? The Australian dollar $ AUD, Is also within 30% of USD $

**Comment by Overtorment** [comment]
@ncoelho probably an elegant UX solution exists for such cases

**Comment by ncoelho** [comment]
Yes, something we can look into if there are many complains. Doesn't seem a huge edge case, as 99.9% of the users will use one local currency.

**Comment by Transisto** [comment]
Should test out how Mycelium was working back then for a good UI implementation.

I supports many relative currency supported at once.

**Comment by BrotherCreamy** [comment]
Most non-US users are constantly having to switch between USD and their local currency.
E.g. Exchange prices are often quoted in USD, most online vendors quote in USD.

Shopify has decent guidelines for currency formatting:
https://polaris.shopify.com/foundations/foundations/formatting-localized-currency
